src/modules/torqueBalancing/README.md

```diff
@@ -45,6 +45,22 @@ Users willing to use the module should follow this list.
 - `kImp`: gains for the impedance (low level) task. The number must match the number of torque controlled joints.
 - `tsat`: saturations to be applied to the output torques. The number must match the number of torque controlled joints. It must be a positive value.
 
+#####Low level torque gains
+The following two optional elements are available at configuration time to specify the low level torque control gains to be used during the balancing control.
+
+- `torqueGains`: specifies either a single file, or a list of key-file pairs containing the low level torque gains. See below for the file format.
+- `defaultTorqueGainsKey`: specifies the default torque gains set to be loaded when the module starts. If only one set of gains has been specified and this option is not set, the only set of gains will be loaded.
+
+It is possible to specify multiple set of torque gains in order to switch them at runtime (see RPC section).
+As an example:
+
+- Single set: `torqueGains torqueGains.ini`
+- Multiple sets: `torqueGains ((set1, torqueGains_first.ini), (set2, torqueGains_second.ini))`
+
+The file contains for each joint a list of values for the proportional, integrative and derivative gains, e.g.
+`l_shoulder_roll     = ("kp", 5.0), ("ki", 0.0),("kd", 4.0)`
+The original gains read from the firmware will be overwritten with the specified gains in the configuration file (it is then possible to specify only the gains that **must** change)
```

Comment:
@francesco-romano  Loading the gains present in the firmware to restore them at the end of the module is involving to much work in your opinion?